testindex/setup_basic.py

```diff
@@ -0,0 +1,30 @@
+import platform
+import subprocess
+
+the_os = platform.dist()[0]
+
+
+def run_cmd(cmd):
+    """Runs a command and waits for it to finish"""
+    process = subprocess.Popen(cmd, shell=True)
+    process.wait()
+    return process.returncode
```

Comment:
I could not think of a use when i wrote this, but considered the possibility of its use later.
I will see if i can include the check for it in the script